src/v/kafka/server/connection_context.cc

```diff
@@ -166,23 +166,18 @@ connection_context::throttle_request(
     if (!delay.first_violation) {
         fut = ss::sleep_abortable(delay.duration, _rs.abort_source());
     }
-    auto track = track_latency(hdr.key);
```

Comment:
Now since we are explicitly recording the latencies that this method was previously attempting to, I've removed this conditional which makes `kafka_rpc` now truly represent what it claims, latency for all requests processed by this server. 

However, now we will have a similar issue. This will also take into consideration the long potential wait times from fetch. 

I'm thinking we rename this metric and modify the conditional to exclude produce / fetch requests, since those metrics are handled by additional code in this PR. Thoughts?

Reply:
So this would mostly be admin requests? If that is the case, that sounds good. It would be good to see any really high latency admin requests coming into the system.

Reply:
It would be like list offsets, metadata, etc requests like that